.github/workflows/workflow.yaml

```diff
@@ -126,16 +130,9 @@ jobs:
           DOCKER_USER: ${{ secrets.DOCKER_USER }}
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
 
-      - uses: 'marvinpinto/action-automatic-releases@v1.0.0'
```

Comment:
removed this, didn't see much value add for pre-release. let me know if we should keep it

Reply:
I thought we added this because there were users interested in running builds off of head? IIRC someone directly asked for this. I wonder if there are download metrics to see if anyone uses it.